Whiskey/Functions/New-WhiskeyContext.ps1

```diff
@@ -119,16 +119,16 @@ function New-WhiskeyContext
 
     $packageJsonPath = Join-Path -Path $buildRoot -ChildPath 'package.json'
     $ignorePackageJsonVersion = $config.ContainsKey('IgnorePackageJsonVersion') -and $config['IgnorePackageJsonVersion']
-    if( -not $config.ContainsKey('Version') -and (Test-Path -Path $packageJsonPath -PathType Leaf) -and -not $ignorePackageJsonVersion )
+    $versionParam = @{}
+    if( $config.ContainsKey( 'VersionPath') )

```

Comment:
Rename this to `VersionFrom`. Rename other usages, too.

Reply:
Done